_includes/portfolio.html

```diff
@@ -0,0 +1,72 @@
+{% if include.portfolio %}
+	{% assign portfolio = include.portfolio %}
+{% else %}
+	{% assign portfolio = page.portfolio.items %}
+{% endif %}
+{% if include.categories %}
+	{% assign categories = include.categories %}
+{% else %}
+	{% assign categories = page.portfolio.categories %}
+{% endif %}
+{% if page.portfolio.button %}
+	{% assign button = page.portfolio.button %}
+{% else %}
+	{% assign button = "Launch Site" %}
+{% endif %}
+
+<div class="portfolio-wrapper showcase">
+	<ul class="filter">
+		<li><a class="active" href="#" data-filter="*">All</a></li>
+		{% for category in categories %}
+			<li><a href="#" data-filter=".{{ category[0] }}">{{ category[1] }}</a></li>
+		{% endfor %}
+	</ul>
+	<ul class="items col4">
+		{% for item in portfolio %}
+		<li class="item {{ item.categories|join: ' ' }}" ata-callback="callPortfolioScripts();" data-detailcontent='
+			<div class="content">
+				{% assign num_images = item.images|size %}
+				<div {% if num_images > 1 %}id="myCarousel" class="carousel slide"{% else %}class="single"{% endif %}>
+					{% if num_images > 1 %}<div class="carousel-inner">{% endif %}
+						{% assign title_num = forloop.index %}
+						{% for image in item.images %}
+							{% if num_images > 1 %}<div class="{% if forloop.first %}active {% endif %}item">{% endif %}
+								<img alt="" width="100%" src="{{ image }}"><a href="{{ image }}" class="fancybox-media" rel="item-{{ title_num }}">
+								<span class="link"></span></a>
+							{% if num_images > 1 %}</div>{% endif %}
+						{% endfor %}
+					{% if num_images > 1 %}
+				</div>
+						<a class="carousel-control left" href="#myCarousel" data-slide="prev"></a>
+						<a class="carousel-control right" href="#myCarousel" data-slide="next"></a>
+					{% endif %}
+				</div>
+			</div>
+			<div class="item-details">
+				<h2>{{ item.heading | markdownify | remove: "<p>" | remove: "</p>" }}</h2>
+				<ul class="item-info">
+					{% for info in item.info %}
+						<li>{% if info[0] != "_blank" %}<span class="lite1">{{ info[0] | markdownify | remove: "<p>" | remove: "</p>" }}: </span>{% endif %}{{ info[1] | markdownify | remove: "<p>" | remove: "</p>" }}</li>
```

Comment:
The whitespace after the `:` (content of  `<span class="lite1">`) isn't intended, is it? Furthermore, the `border-bottom` spans over a whitespace after the element for some reason, I don't know why this happens, but it looks a bit odd.